unstructured/partition/email.py

```diff
@@ -252,16 +265,18 @@ def partition_email(
     max_partition
         The maximum number of characters to include in a partition. If None is passed,
         no maximum is applied. Only applies if processing the text/plain content.
+    min_partition
+        The minimum number of characters to include in a partition. Only applies if
+        processing the text/plain content.
```

Comment:
nit/curious: why move this here? before it looked like we were preserving the docstring args order per the order in the signature, but now not so?

Reply:
Yeah updated this to preserve the ordering from signature. I think that probably got added where it was so that it would be next to `max_partition`. We've tended to add new kwargs to the end to avoid breaking things for people who are using argument order instead of kwarg names (though admittedly it would be weird to use _all_ these kwargs while depending on the order).